lib/tpm2_alg_util.c

```diff
@@ -115,7 +115,7 @@ static void tpm2_alg_util_for_each_alg(alg_iter iterator, void *userdata) {
     }
 }
 
-static alg_parser_rc handle_sym_common(const char *ext, TPMT_SYM_DEF_OBJECT *s) {
+static alg_parser_rc handle_sym_common(const char *ext, TPMT_SYM_DEF_OBJECT *s, bool is_parent) {
 
```

Comment:
Is this a rebase left over? I think for the feature you are enabling this change doesn't add up. All changes to this file fall in the same camp.